docs_user/modules/proc_adopting-the-compute-service.adoc

```diff
@@ -118,31 +101,33 @@ spec:
             standalone:
               customServiceConfig: |
                 [DEFAULT]
-                host = standalone.localdomain
+                host = <hostname>
                 [workarounds]
                 disable_compute_service_check_for_ffu=true
 ----
++
+* Replace <hostname> with the hostname of the node that is running the `ironic` Compute driver in the source cloud.
 
-. Wait for {compute_service} control plane services' custom resources (CRs) to become ready:
+. Wait for the CRs for the Compute control plane services to be ready:
 +
 ----
-oc wait --for condition=Ready --timeout=300s Nova/nova
+$ oc wait --for condition=Ready --timeout=300s Nova/nova
 ----
 +
-The local Conductor services will be started for each cell, while the superconductor runs in `cell0`.
-Note that `disable_compute_service_check_for_ffu` is mandatory for all imported Nova services, until the external data plane is imported, and until Nova Compute services fast-forward upgraded. For more information, see xref:adopting-compute-services-to-the-data-plane_data-plane[Adopting Compute services to the {rhos_acro} data plane] and xref:performing-a-fast-forward-upgrade-on-compute-services_data-plane[Performing a fast-forward upgrade on Compute services].
+The local Conductor services are started for each cell, while the superconductor runs in `cell0`.
```

Comment:
Looking at the Compute doc, it would seem that superconductor is two words:
https://docs.redhat.com/en/documentation/red_hat_openstack_platform/17.1/html-single/scaling_deployments_with_compute_cells/index#assembly_multi-cell-overcloud-deployments_cellsv2

Unless this is the name of some file or actual process.

Reply:
we use it both ways 
the downstream docs are not the source of truth to reference
instead we should look to the upstream docs

https://github.com/openstack/nova/blob/master/doc/source/admin/cells.rst#api-database

we typically refer to this as one word

but we also spell it out as two 

https://github.com/openstack/nova/blob/1191a49a65dce694b12df26bb5316b0a816b278e/doc/source/admin/cells.rst?plain=1#L179